test/bids_examples_eeg_rest_fmri.jsonld

```diff
@@ -0,0 +1,237 @@
+        }
+      },
+      "internalIdentifier": "Studied state sub-32",
+      "lookupLabel": "Studied state sub-32"
+    },
+    {
+      "@id": "https://openminds.ebrains.eu/instances/unitOfMeasurement/year",
+      "@type": "https://openminds.ebrains.eu/controlledTerms/UnitOfMeasurement",
+      "name": "year"
+    },
+    {
+      "@id": "_:000002",
+      "@type": "https://openminds.ebrains.eu/core/Subject",
+      "internalIdentifier": "sub-32",
+      "lookupLabel": "sub-32",
+      "studiedState": [
+        {
+          "@id": "_:000000"
+        }
+      ]
+    },
+    {
+      "@id": "_:000003",
+      "@type": "https://openminds.ebrains.eu/core/SubjectState",
+      "age": {
+        "@type": "https://openminds.ebrains.eu/core/QuantitativeValue",
+        "unit": {
+          "@id": "https://openminds.ebrains.eu/instances/unitOfMeasurement/year"
+        }
+      },
+      "internalIdentifier": "Studied state sub-35",
+      "lookupLabel": "Studied state sub-35"
+    },
+    {
+      "@id": "_:000004",
+      "@type": "https://openminds.ebrains.eu/core/Subject",
+      "internalIdentifier": "sub-35",
+      "lookupLabel": "sub-35",
+      "studiedState": [
+        {
+          "@id": "_:000003"
+        }
+      ]
+    },
+    {
+      "@id": "_:000005",
+      "@type": "https://openminds.ebrains.eu/core/SubjectState",
+      "age": {
+        "@type": "https://openminds.ebrains.eu/core/QuantitativeValue",
+        "unit": {
+          "@id": "https://openminds.ebrains.eu/instances/unitOfMeasurement/year"
+        }
+      },
+      "internalIdentifier": "Studied state sub-36",
+      "lookupLabel": "Studied state sub-36"
+    },
+    {
+      "@id": "_:000006",
+      "@type": "https://openminds.ebrains.eu/core/Subject",
+      "internalIdentifier": "sub-36",
+      "lookupLabel": "sub-36",
+      "studiedState": [
+        {
+          "@id": "_:000005"
+        }
+      ]
+    },
+    {
+      "@id": "_:000007",
+      "@type": "https://openminds.ebrains.eu/core/Person",
+      "familyName": "Deligianni",
+      "givenName": "F."
+    },
+    {
+      "@id": "_:000008",
+      "@type": "https://openminds.ebrains.eu/core/Person",
+      "familyName": "Centeno",
+      "givenName": "M."
+    },
+    {
+      "@id": "_:000009",
+      "@type": "https://openminds.ebrains.eu/core/Person",
+      "familyName": "Carmichael",
+      "givenName": "D.W."
+    },
+    {
+      "@id": "_:000010",
+      "@type": "https://openminds.ebrains.eu/core/Person",
+      "familyName": "Zhang",
+      "givenName": "G.H."
+    },
+    {
+      "@id": "_:000011",
+      "@type": "https://openminds.ebrains.eu/core/Person",
+      "familyName": "Clark",
+      "givenName": "C.A."
+    },
+    {
+      "@id": "_:000012",
+      "@type": "https://openminds.ebrains.eu/core/Person",
+      "familyName": "Clayden",
+      "givenName": "J.D."
+    },
+    {
+      "@id": "_:0000013",
+      "@type": "https://openminds.ebrains.eu/core/FileRepository"
```

Comment:
why doesn't the FileRepository have any properties? e.g. `iri`?

Reply:
What would be an `IRI` of a file repository? The `IRI` of the main folder?

Reply:
yes

Reply:
Done, it is in pull request #41.